smoketest.bash

```diff
@@ -119,6 +137,40 @@ setupUserHosts() {
 }
 setupUserHosts
 
+openBrowserTabs() {
+    # TODO find a way to use 'podman wait --condition=healthy $containerId' instead of polling with curl
+    set +xe
+    local urls=()
+    for file in "${FILES[@]}"; do
+        yaml="$(yq '.services.* | [{"host": .hostname, "ports": .ports}]' "${file}")"
+        length="$(echo "${yaml}" | yq 'length')"
+        for (( i=0; i<"${length}"; i+=1 ))
+        do
+            host="$(echo "${yaml}" | yq ".[${i}].host" | grep -v null)"
+            port="$(echo "${yaml}" | yq ".[${i}].ports[0]" | grep -v null | cut -d: -f1)"
+            if [ -n "${host}" ] && [ -n "${port}" ]; then
+                urls+=("http://${host}:${port}")
+            fi
+        done
+    done
+    set -xe
+    echo "Service URLs:" "${urls[@]}"
```

Comment:
my logs show the service urls are `http://s3:9001` and `http://cryostat3:8181`, which I believe should be `http://localhost:8181`? Should it also include the links to `grafana` and `db-viewer`?

Reply:
`s3:9001` and `cryostat3:8181` are correct, but this assumes that the "userhosts" `LD_PRELOAD` setup is working properly. If that isn't configured then you won't be able to resolve the hostnames `s3` and `cryostat3`, which are supposed to just resolve to `localhost`.

`grafana` and `db-viewer` should be there if you enable those with `smoketest.bash -g` and `smoketest.bash -X` respectively.

Reply:
```bash
$ cat ~/.hosts 
localhost db
localhost db-viewer
localhost grafana
localhost jfr-datasource
localhost vertx-fib-demo-1
localhost quarkus-test-agent
localhost s3
localhost cryostat3
```